jetbrains-core/resources/icons/resources/sqs/SqsQueue_dark.svg

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="18px" height="17px" viewBox="0 0 18 17" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
16x16

Reply:
I see, should I manually change the width and height?

Reply:
It is fixed.